modules/twinklearv.js

```diff
@@ -812,6 +768,68 @@ Twinkle.arv.callback.evaluate = function(e) {
 	}
 };
 
+Twinkle.arv.callback.getAivReasonOnlyWikitext = function(input) {
```

Comment:
Consider renaming to getAivReasonWikitext, or a different name, or adding a comment. I don't understand the "only" part.

Reply:
Thanks - I didn't like that name and thought it was too long but somehow it didn't click that "only" was completely redundant.